cli_tools/common/assert/doc.go

```diff
@@ -0,0 +1,19 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// Package assert provides convenience functions that help a method or constructor check whether
+// it was invoked correctly (whether its preconditions have been met).
+//
+// When the assertion fails, a panic occurs, allowing bugs to be found in a fail-fast manner.
+package assert
```

Comment:
What's the purpose of this file?

Reply:
It's just for documentation: https://blog.golang.org/godoc